app/models/item.rb

```diff
@@ -1,6 +1,6 @@
 class Item < ApplicationRecord
     validates_presence_of :name, :description, :unit_price
-    enum status: { Enabled: 1, Disabled: 0 }
```

Comment:
just curious about why the enums were changed, I saw there was a new migration that added a default setting to an item so perhaps that was it? I'll ask!